libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -14,6 +14,8 @@
 #define INS_MAX_INSTANCES 3
 #define INS_MAX_BACKENDS  6
 #define INS_VIBRATION_CHECK_INSTANCES 2
+#define XYZ_AXIS_COUNT    3
```

Comment:
this will collide with the one in AP_GyroFFT.h

Reply:
Thanks fixed